taxonomies/categories.txt

```diff
@@ -1117,6 +1210,36 @@ ro:Băuturi spirtoase
 zh:硬酒
 wikidata:en:Q2832208
 
+<en:Alcoholic beverages
+en:Spirit drinks
+
+<en:Spirit drinks
+en:Spanish spirit drinks
+
+<en:Spanish spirit drinks
+es:Ronmiel de Canarias
+protected_name_file_number:en: PGI-ES-01943
+protected_name_type:en: gi
```

Comment:
protected_name_type:en: gi --> should be pgi
(also in may other entries)

Reply:
it was actually GI